src/ui/localization/translator.ts

```diff
@@ -0,0 +1,128 @@
+import en from "./locale/en.json";
+import es from "./locale/es.json";
+import it from "./locale/it.json";
```

Comment:
should we make a barrel import here to have something like `import { en, es, it } from "./locale" ?

Reply:
they are different files, I don't see the need personally.
Also they are json so I wouldn't add typescript export just for that.